app/src/sharedTest/java/org/oppia/android/app/onboarding/OnboardingFragmentTest.kt

```diff
@@ -223,9 +223,19 @@ class OnboardingFragmentTest {
     }
   }
 
+  @Test
+  fun testNextButtonSpotlight_setToShowOnStartup_checkSpotlightIsShown() {
+    launch(OnboardingActivity::class.java).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(withText(R.string.onboarding_next_button_spotlight_hint)).check(matches(isDisplayed()))
+    }
+  }
+
   @Test
   fun testOnboardingFragment_checkDefaultSlide_clickSkipButton_shiftsToLastSlide() {
     launch(OnboardingActivity::class.java).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(withId(R.id.close_target)).perform(click())
```

Comment:
Suggest moving this to a helper to both simplify the changed tests here & to provide context (e.g. ``skipNextButtonSpotlight``).

Reply:
using skipNextSlideButtonSpotlight as suggested.